### PR TITLE
DOCSP-6927: Atlas Full Text Search Promo

### PR DESCRIPTION
**Pull request by rkumar-mongo**
@kay-kim would appreciate a quick headcheck that I haven't made a critical error somewhere in here.

### CODE REVIEW

**Comment by kay-kim** [comment]
@rkumar-mongo  - did you verify the wording change with Timothy Frietas (the PM)

File: source/includes/promo-atlas-fts.rst [hunk 1]
```diff
@@ -0,0 +1,14 @@
+.. admonition:: Atlas Full Text Search
+   :class: note
+
+   `MongoDB Atlas <https://www.mongodb.com/cloud/atlas?jmp=docs>`_
+   Full-Text Search and Full-Text Indexes leverage Apache Lucene to
```
Comment (timfrietas) [inline source/includes/promo-atlas-fts.rst]: Maybe just `Full-Text Search Indexes` instead of `Full-Text Search and Full-Text Indexes`

**Comment by rkumar-mongo** [comment]
Suggestion integrated. @kay-kim if there are no other comments I can merge this up.

**Comment by kay-kim** [comment]
@rkumar-mongo  -- if you could just tweak to add the currency USD, then ship it.

**Pull request merged**